tests/wallet/test_wallet.c

```diff
@@ -1,10 +1,27 @@
 // Copyright 2020 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+#include <stdio.h>
+#include <unistd.h>  // for Linux sleep()
+
+#include "client/api/restful/faucet_enqueue.h"
 #include "core/address.h"
+#include "core/utils/macros.h"
 #include "unity/unity.h"
 #include "wallet/wallet.h"
 
+#define TEST_WITH_PRIVATE_TANGLE 0
+
+// API endpoint
+#define NODE_HOST "localhost"
+#define NODE_PORT 14265
+#define NODE_IS_HTTPS 0
+
+// faucet endpoint
+#define FAUCET_HOST "localhost"
+#define FAUCET_PORT 8091
+#define FAUCET_IS_HTTPS 0
```

Comment:
Should a `wallet` module also have a common `test_config.h` file? Or maybe whole library could have a common one?